index.js

```diff
@@ -721,6 +721,7 @@ class GitlabScm extends Scm {
                 });
 
                 return {
+                    id: author.id,
```

Comment:
Need to update this similar to https://github.com/screwdriver-cd/scm-github/pull/217
```
id: user.data.id.toString()
```